packages/sol-meta/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "lib",
+        "rootDir": "."
+    },
+    "include": [
+        "./src/**/*",
+        "./test/**/*",
+        "../typescript-typings/types/solidity-parser-antlr/*"
```

Comment:
Why not use typeRoots as in other places?

Reply:
IDK. @fabioberger What is the correct way to get typings to work?